content/docs/command-reference/exp/push.md

```diff
@@ -60,8 +60,14 @@ This command will also try to [push](/doc/command-reference/push) all
   (first parents) starting from the `--rev` baseline. Give a negative value to
   include all first-parent commits (similar to `git log -n`).
 
-- `-f`, `--force` - rewrite the `experiment` commit if it already exists in the
-  Git remote. Equivalent to `git push --force` (rewrites history)
+- `-f`, `--force` - rewrite the experiment if it already exists in the Git
+  remote.
+
+  <admon type="warn">
+
+  Similarly to `git push --force`, this rewrites the remote history.
+
+  </admon>
```

Comment:
Similar to https://github.com/iterative/dvc.org/pull/4200#discussion_r1056670149, this may be overly alarming since it sounds like it could rewrite histories of branches instead of only exp refs.

```suggestion

```